corehq/ex-submodules/phonelog/migrations/0012_server_date_not_null.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-06-30 15:48
+from __future__ import unicode_literals
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0011_partition_devicelogentry'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='devicereportentry',
+            name='server_date',
+            field=models.DateTimeField(db_index=True, default=datetime.datetime(2018, 6, 30, 0, 0, 0, 0)),
```

Comment:
Ah. I meant to comment on this. This shouldn't be hit because it's a new table, but by making it that far ahead i can easily go check after the migration.

in the current device log table on NIC we don't have any records taht are null. on staging we had a lot, but I think that would have been an earlier bug. that shouldn't happen any longer